fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonUtils.xtend

```diff
@@ -259,33 +259,42 @@ class PythonUtils {
 		«ENDIF»
 		«ENDIF»
 		«IF !cmd.argout.type.voidType»    dtype_out=«cmd.argout.type.pythonTypeHL», 
-		«IF !cmd.argout.description.empty»    doc_out="«cmd.argout.description.oneLineString»", 
-		«ENDIF»
-		«ENDIF»
-		«IF !cmd.description.empty»    description="«cmd.description.oneLineString»",
 		«ENDIF»
     «setAttrPropertyHL("display_level", cmd.displayLevel, false)»
     «setAttrPropertyHL("polling_period", cmd.polledPeriod, false)»
 		«IF cmd.hasCommandArg»    )
 		«ENDIF»
+		«ENDIF»
 	«ENDIF»
-«ENDIF»
     @DebugIt()
     def «cmd.methodName»(self«IF !cmd.argin.type.voidType», argin«ENDIF»):
         «IF cls.description.filestogenerate.toLowerCase.contains("protected regions")»
-            «IF !cmd.argout.type.voidType»
-                «protectedAreaHL(cls, cmd.name, cmd.argout.type.defaultValueReturnHL, false)»
-            «ELSE»
-                pass
-            «ENDIF»
+        «openProtectedAreaHL(cls,cmd.name)»
+        """
+        «cmd.description.commentCmdParamMultiLines»
+        «IF !cmd.argin.type.voidType»
+        
+        :param argin: «cmd.argin.type.pythonTypeHL»
+        «cmd.argin.description.commentCmdParamMultiLines»
+        «ENDIF»
+        «IF !cmd.argout.type.voidType»
+        
+        :return:«cmd.argout.type.pythonTypeHL»
+        «cmd.argout.description.commentCmdParamMultiLines»
         «ELSE»
-            «IF !cmd.argout.type.voidType»
-                return «cmd.argout.type.defaultValueTestHL»
-            «ELSE»
-                pass
-            «ENDIF»
+        
+        :return:None
+        «ENDIF»
+        """
+        «cmd.argout.type.defaultValueReturnHL»
+        «closeProtectedAreaHL(cls,cmd.name)»
+        «ELSE»
+        «IF !cmd.argout.type.voidType»
+        return «cmd.argout.type.defaultValueTestHL»
+        «ELSE»
+        pass
+        «ENDIF»
         «ENDIF»
-
```

Comment:
Blank line should be retained.
Please run `flake8` on the generated file - there are many places with trailing spaces, bad indentation, and too many or too few blank lines.
The warnings about unused imports can probably be ignored - they might be used when someone implements a device.

Reply:
Run flake8 on generated code and tried minimizing the warnings.
Link to commit id:
https://github.com/ska-telescope/pogo/commit/358dca47fff64e3f35d4a355d773ef6821eb198c
https://github.com/ska-telescope/pogo/commit/1ff58be1d801d6cfca9cc257e606056f097ea613